src/Compilers/CSharp/Test/Syntax/Parsing/ParserRegressionTests.cs

```diff
@@ -83,24 +81,7 @@ public void c024928()
         [Fact]
         public void TestBinary()
```

Comment:
Yep, I'll pay attention during merge.  Fortunately Git should flag it as a conflict.